### PR TITLE
Miniwdl oop

### PR DESCRIPTION
**Pull request by Th3nn3ss**
Draft to use miniwdl to implement an object oriented version of the wdl2cwl converter.

1. Convert a single wdl file to cwl by hand without loops, if conditions or classes.
    - Handled inputs
    - Handled outputs
    - Handled command section
    - Handled runtime section

### CODE REVIEW

File: wdl2cwl/tests/oop_cwl_files/bowtie_1.cwl [hunk 1]
```diff
@@ -0,0 +1,60 @@
+class: CommandLineTool
+id: Bowtie
```
Comment (mr-c) [inline wdl2cwl/tests/oop_cwl_files/bowtie_1.cwl:2]: Does this file match the `main.py` output?

Comment (Th3nn3ss) [reply wdl2cwl/tests/oop_cwl_files/bowtie_1.cwl:2]: Yes. But the requirements is incomplete.

Am still exploring the command section from the WDL file.

File: wdl2cwl/main_oop.py [hunk 2]
```diff
@@ -0,0 +1,584 @@
+import os
+from typing import List, Union, Optional, Callable
+import WDL
```
Comment (mr-c) [inline wdl2cwl/main_oop.py:3]:
```suggestion
import WDL  # type: ignore[import]
```

**Review by kinow — APPROVED** [review]
:drum: great job @Th3nn3ss !+1

**Pull request merged**